content/blog/2019-01-03-ozunconf18.md

```diff
@@ -0,0 +1,54 @@
+---
+slug: "ozunconf18"
+title: Continuing to Grow Community Together at ozunconf, 2018
+authors:
+  - name: Nicholas Tierney
+    url: https://www.njtierney.com
+    twitter: nj_tierney
+# twitterImg: img/blog-images/2019-01-03-ozunconf18/fun-photo.jpg
+date: 2019-01-03
```

Comment:
date: 2019-01-08
in YAML, path to photos, filename etc pls

Reply:
OK no worries - do I need to change the filename to be "2019-01-08..." intead of "2019-01-03..." ?

Reply:
yes please

Reply:
OK done!